docs/reference/commandline/cp.md

```diff
@@ -89,6 +89,25 @@ you must be explicit with a relative or absolute path, for example:
 
     `/path/to/file:name.txt` or `./file:name.txt`
 
+## Examples
+
+Copy a local file into container
+```console
+$ docker cp ./some_file CONTAINER:/work 
+```
+
+Copy files from container to local path
+```console
+$ docker cp CONTAINER:/var/logs/ /tmp/app_logs
+```
+
+Copy a file from container to stdout. Please note `cp` command produces a tar stream
+```console
+$ docker cp CONTAINER:/var/logs/app.log - | tar x -O | grep "ERROR"
+```
+
+## Corner cases
```

Comment:
Can you change this to a H3? The `yamldocs` we generate for docs.docker.com collect everything under the `H2 Description` and `H2 Examples` sections. Putting this as a H2 causes it to be skipped (but it will be included if it's a H3 _within_ the "examples" section

 ```suggestion
### Corner cases
```